File: java/1030A.java
// 1030 A. In Search of an Easy Problem

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        boolean hard = false;

        while (n > 0){
            int i = sc.nextInt();
            if(i==1){
                hard = true;
            }
            n--;
        }
        if(hard) System.out.println("HARD");
        else System.out.println("EASY");
    }
}

File: cpp/1030A.cpp
// 1030 A. In Search of an Easy Problem

#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;
    bool hard = false;
    
    while (n--){
        int i;
        cin >> i;

        if(i == 1) {
            hard = true;
        }
    }
    
    if(hard) cout << "HARD";
    else cout << "EASY";
}
